packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -70,6 +70,75 @@ const DEFAULT_SCHEDULE = {
   timezone: moment.tz.guess(),
 }
 
+const ReportReceivers = decorate([
+  provideState({
+    initialState: () => ({
+      receiver: '',
+    }),
+    effects: {
+      linkState,
+      addReceiver() {
```

Comment:
IMO we can say `add()` here since we're already in the context of `ReportReceivers`.